frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -245,7 +248,9 @@ export const OperandStatusIconAndText: React.FunctionComponent<OperandStatusIcon
   return iconAndText;
 };
 
-export const OperandTableRow: React.FC<OperandTableRowProps> = ({ obj, index, key, style }) => {
+export const OperandTableRow = connectToFlags<OperandTableRowProps & WithFlagsProps>(
```

Comment:
Should we use connectToFlags on OperandList and pass the flags through to this component? I think in most cases, where possible, we like to connect higher up and pass the state down to children.

Reply:
Wouldn't we have to make changes to Table to configure it to pass the flags props to the row? IMO that would be more complicated than this.